packages/frontend/src/components/DataViz/config/CartesianChartConfiguration.tsx

```diff
@@ -32,56 +25,27 @@ export const CartesianChartConfig = ({
 
     return (
         <Stack spacing="xs" mb="lg">
-            <Group spacing="xs">
-                <Title order={5} fz="sm" c="gray.9">
-                    Data
-                </Title>
-                <ActionIcon>
-                    <MantineIcon
-                        onClick={() =>
-                            setIsFieldConfigurationOpen(
-                                !isFieldConfigurationOpen,
-                            )
-                        }
-                        icon={
-                            isFieldConfigurationOpen
-                                ? IconChevronDown
-                                : IconChevronRight
-                        }
+            <Tabs color="gray" defaultValue="data" keepMounted>
+                <Tabs.List>
+                    <Tabs.Tab value="data">Data</Tabs.Tab>
+                    <Tabs.Tab value="styling">Styling</Tabs.Tab>
+                </Tabs.List>
+
+                <Tabs.Panel value="data" pt="xs">
+                    <CartesianChartFieldConfiguration
+                        actions={actions}
+                        columns={columns}
+                        selectedChartType={selectedChartType}
                     />
-                </ActionIcon>
-            </Group>
-
-            {isFieldConfigurationOpen && (
-                <CartesianChartFieldConfiguration
-                    actions={actions}
-                    columns={columns}
-                    selectedChartType={selectedChartType}
-                />
-            )}
+                </Tabs.Panel>
 
-            <Divider my="sm" />
-
-            <Group spacing="xs">
-                <Title order={5} fz="sm" c="gray.9">
-                    Styling
-                </Title>
-                <ActionIcon>
-                    <MantineIcon
-                        onClick={() => setIsStylingOpen(!isStylingOpen)}
-                        icon={
-                            isStylingOpen ? IconChevronDown : IconChevronRight
-                        }
+                <Tabs.Panel value="styling" pt="xs">
+                    <CartesianChartStyling
```

Comment:
there will be more changes in the future re: 
- where do we display `stacked` config? under groupby in configuration or should it be a tab in itself? 

@PriPatel will cover this in next steps of redesign ⬆️